src/ngx_http_lua_headers.c

```diff
@@ -475,6 +477,22 @@ ngx_http_lua_ngx_resp_get_headers(lua_State *L)
         return luaL_error(L, "no request object found");
     }
 
+    ctx = ngx_http_get_module_ctx(r, ngx_http_lua_module);
+    if (ctx == NULL) {
+        return luaL_error(L, "no ctx found");
+    }
+
+    if (!ctx->headers_set) {
+        rc = ngx_http_lua_set_content_type(r);
+        if (rc != NGX_OK) {
+            return luaL_error(L,
+                              "failed to set default content type: %d",
+                              (int) rc);
+        }
+        
```

Comment:
This line has trailing spaces. Better use the `ngx-releng` tool to check such things. For this case, `ngx-releng` reports the following error (the error message was actually in red):

```
src/ngx_http_lua_headers.c:
found unnecessary tail space
492:
```